messaging/kafka-processor/pom.xml

```
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <properties>
+        <!-- Need to be this version, I wasn't able to reproduce QUARKUS-5178 on newer versions-->
+        <quarkus-kafka-streams-processor>2.0.1</quarkus-kafka-streams-processor>
```

```suggestion
        <quarkus-kafka-streams-processor.version>2.0.1</quarkus-kafka-streams-processor.version>
```
Plus where the property is used ...
Motivation: to be in sync with naming in root pom.xml

---

Good point, fixed